APE13.rst

```diff
@@ -0,0 +1,450 @@
+            which correspond to a single sky coordinate object). The second
+            element is  either a string keyword argument name or a positional
+            index for the corresponding class from
+            ``world_axis_object_metadata``.
+            """
+
+        @property
+        def world_axis_object_metadata(self):
+            """
+            A dictionary with each key being a string key from
+            ``world_axis_object_components``, and the value being a 2-tuple.
+            The first element of the tuple must be a string specifying the
+            fully-qualified name of a class, which will specify the actual
+            Python object to be created. The second tuple element must be a
+            dictionary with the keyword arguments required to initialize the
+            class.
+
+            Note that we don’t require the classes to be Astropy classes since
+            there is no guarantee that Astropy will have all the classes to
+            represent all kinds of world coordinates. Furthermore, we recommend
+            that the output be kept as human-readable as possible.
+            """
+
+We now take a look at an example of use of ``world_axis_object_components`` with
+``world_axis_object_metadata``. An example output from both methods on the same
+WCS object is:
+
+.. code-block:: python
+
+    >>> wcs.world_axis_object_components
+    [('skycoord', 'ra'), ('time', 0), ('skycoord', 'dec')]
+    >>> wcs.world_axis_object_metadata
+    {'skycoord': (‘astropy.coordinates.SkyCoord’,
+                  {'frame': 'fk5', 'equinox':'J2005'}),
+     'time': (‘astropy.time.Time’, {'scale': 'tai'})}
+
+This indicates that the first and third world axis can be used to instantiate an
+Astropy ``SkyCoord`` object with ``ra=`` set to the first world axis, and
+``dec=`` set to the third axis, and the ``frame=fk5`` and ``equinox=J2005``
+arguments, while the second world axis can be used to instantiate an Astropy
+``Time`` object as the first positional argument, and with the ``scale=tai``
+keyword argument. Note that the coordinate frame classes could be custom
+sub-classes if needed.
+
+Low-level API examples
+^^^^^^^^^^^^^^^^^^^^^^
+
+**Simple 1D spectrum** - 1D mapping from pixel to wavelength:
+
+.. code-block:: python
+
+    wcs.axis_correlation_matrix = [[True]]
+    wcs.world_axis_units = [‘angstrom’]
+    wcs.world_axis_physical_type = [‘spectral’]
+    wcs.world_axis_object_components = [(‘spec’, 0)]
+    wcs.world_axis_object_classes  = {‘spec’:(‘astropy.units.Wavelength’:
+                                              {‘airorvacwl’: ‘air’})}
+
+**Simple 2D image mapping** where the axis are just RA and Dec (in FITS-WCS this would be CAR)
+
+.. code-block:: python
+
+    wcs.axis_correlation_matrix = [[True, False], [False, True]]
+    wcs.world_axis_units = [‘deg’, ‘deg’]
+    wcs.world_axis_physical_type = [‘pos.eq.ra’, ‘pos.ra.dec’]
```

Comment:
Typo here: `pos.ra.dec` shpould be `pos.eq.dec`.

Reply:
Thanks!